python/dataset_plotting.py

```diff
@@ -238,30 +238,39 @@ def plot_image(input_data, axes=None, contours=False, plot=True):
                 raise RuntimeError("Dimensions of x and y arrays to not match "
                                    "that of the Value array.")
 
+        if contours:
+            plot_type = 'contour'
+        else:
+            plot_type = 'heatmap'
+
+        title = values[0].name
+        if logcb:
+            title = "log(" + title + ")"
```

Comment:
You are using `log10`, so I'd be explicit here. In many cases `log` is read as natural logarithm.